.travis.yml

```diff
@@ -24,7 +24,7 @@ before_install:
 # By setting anything to org.apache.activemq.artemis.core.io.aio.AIOSequentialFileFactory.DISABLED we are disabling libaio loading on the testsuite
 script: 
 - set -e
-- mvn -Dorg.apache.activemq.artemis.core.io.aio.AIOSequentialFileFactory.DISABLED=AnythingNotNull -Pfast-tests -Pextra-tests -Ptests-CI -B install -q
+- mvn -Dorg.apache.activemq.artemis.core.io.aio.AIOSequentialFileFactory.DISABLED=AnythingNotNull -Djdk8-errorprone -Pfast-tests -Pextra-tests -Ptests-CI -B install -q
```

Comment:
didn't you mean -Pjdk8-errorprone?


-P

Reply:
No, as that would enable the JDK8 arguments on 11+ and break them. The profile can be activated explicitly with -P or by JDK8+property. Using the property as here stops it from actually adding the JDK-8 specific bits on JDK11+ when run there (since the job script is shared across all 3) which would break.

Reply:
@gemmellr thanks for the clarification.